app_management/initial_startup.rb

```diff
@@ -23,33 +23,28 @@
 $LOAD_PATH.unshift app_mgmt_dir
 
 require 'json'
-require 'utils/handlers'
-require 'utils/simple_logger'
+require_relative 'utils/handlers'
+require_relative 'utils/simple_logger'
 
-def get_handler_list
-  if !ENV['BLUEMIX_APP_MGMT_ENABLE'].nil?
-    handlers = ENV['BLUEMIX_APP_MGMT_ENABLE'].downcase.split('+').map(&:strip)
-  else
-    handlers = nil
-  end
-  return handlers
+def handler_list
+  ENV['BLUEMIX_APP_MGMT_ENABLE']&.downcase&.split('+')&.map(&:strip)
```

Comment:
I think this would be easiest to read if it was:

```
if !ENV['BLUEMIX_APP_MGMT_ENABLE'].nil?
    ENV['BLUEMIX_APP_MGMT_ENABLE'].downcase.split('+').map(&:strip)
  else
    nil
 end
```